### PR TITLE
Adding debuggin tool for workload

### PR DESCRIPTION
**Pull request by samcake**
- FIx a bug with ResourceBuffer binding in gl45 backend.
- Introduce a job to represent debugging information visually about the workload/space
- Expose the Workload engine COnfig to the js api to be able to access it live from the api

### CODE REVIEW

File: libraries/workload/src/workload/Space.h [hunk 1]
```diff
@@ -45,6 +45,8 @@ class Space {
         Sphere sphere;
         uint8_t region { REGION_UNKNOWN };
         uint8_t prevRegion { REGION_UNKNOWN };
+        uint16_t _padding;
+        uint32_t _paddings[3];
```
Comment (samcake) [inline libraries/workload/src/workload/Space.h]: Padded to be a multiple of a  4 scalars for gpu  convenience

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7612/app_12527_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7612/HighFidelity-Beta-PR12527-d91fb543f33a37cb9489bed0d0779032fbf21895.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7612/HighFidelity-Beta-PR12527-d91fb543f33a37cb9489bed0d0779032fbf21895.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7616/app_12527_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7617/app_12527_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7617/HighFidelity-Beta-PR12527-94fa2ef6a0f86b83a458c7a3540abc2c7cec938e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7617/HighFidelity-Beta-PR12527-94fa2ef6a0f86b83a458c7a3540abc2c7cec938e.exe

**Pull request merged**